web/crux/prisma/seeds/demo-seeder.ts

```diff
@@ -0,0 +1,120 @@
+import { Prisma, PrismaClient } from '@prisma/client'
+import { version } from './data/demo/version'
+import { product } from './data/demo/product'
+import { registry } from './data/demo/registry'
+import { images } from './data/demo/images'
+import { DemoConstants } from './consts'
+import { getContainerConfig } from './data/demo/container_config'
+
+const prisma = new PrismaClient()
+
+enum TableNames {
+  Product,
+  Version,
+  Registry,
+  Image,
+  ContainerConfig,
+}
+
+async function main() {
+  const commandLineArgs = process.argv.slice(2)
+  const userId = commandLineArgs[0]
+  const teamId = commandLineArgs[1]
+
+  if (!userId || !teamId) {
+    console.error('Missing command line parameters, try npm run demo-seed "userId" "teamId"')
```

Comment:
Please validate the user and team, if Team doesn't exist we'll get an exception, it would be cleaner if we can give back a proper error message to the user. Check other edge cases.